File: core/src/main/java/com/gmail/marcosav2010/communicator/packet/packets/PacketShutdown.java
package com.gmail.marcosav2010.communicator.packet.packets;

import com.gmail.marcosav2010.communicator.packet.StandardPacket;
import com.gmail.marcosav2010.communicator.packet.wrapper.PacketDecoder;
import com.gmail.marcosav2010.communicator.packet.wrapper.PacketEncoder;
import lombok.NoArgsConstructor;

@NoArgsConstructor
public class PacketShutdown extends StandardPacket {

    @Override
    protected void encodeContent(PacketEncoder out) {
    }

    @Override
    protected void decodeContent(PacketDecoder in) {
    }
}

File: core/src/main/java/com/gmail/marcosav2010/communicator/packet/packets/PacketShutdown.kt
package com.gmail.marcosav2010.communicator.packet.packets

import com.gmail.marcosav2010.communicator.packet.StandardPacket
import com.gmail.marcosav2010.communicator.packet.wrapper.PacketDecoder
import com.gmail.marcosav2010.communicator.packet.wrapper.PacketEncoder

class PacketShutdown : StandardPacket() {

    override fun encodeContent(out: PacketEncoder) {}
    override fun decodeContent(`in`: PacketDecoder) {}
}